clientconn.go

```diff
@@ -639,7 +638,7 @@ func (ac *addrConn) resetTransport(closeTransport bool) error {
 			t.Close()
 		}
 		sleepTime := ac.dopts.bs.backoff(retries)
-		timeout := minConnectTimeout
+		timeout := internal.MinConnectTimeout
```

Comment:
Please still leave minConnectTimeout in grpc package. You only need to add a function in "internal" to set its value in grpc's init.